sqlstore/src/it/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateServiceRecordDaoIT.java

```diff
@@ -73,45 +74,47 @@ public void setup() throws IOException {
     emptySR.setId(2L);
     Mockito.when(instrumentDao.get(ArgumentMatchers.anyLong())).thenReturn(emptySR);
   }
-  
+
   @Test
   public void testSaveNew() throws IOException {
+
     String title = "New Record 1";
     Long newId = dao.save(makeServiceRecord(title));
-    
+
     ServiceRecord savedRec = dao.get(newId);
     assertEquals(title, savedRec.getTitle());
   }
-  
+
   private ServiceRecord makeServiceRecord(String title) {
     ServiceRecord rec = new ServiceRecord();
     rec.setTitle(title);
-    rec.setInstrument(emptySR);
+    instrumentService.addServiceRecord(rec, emptySR);
     rec.setServiceDate(new java.util.Date());
     rec.setServicedByName("Test Person");
     return rec;
   }
-  
+
   @Test
   public void testSaveEdit() throws IOException {
     ServiceRecord rec = dao.get(1L);
     String newTitle = "ChangedTitle";
     rec.setTitle(newTitle);
     Instrument sr = Mockito.mock(Instrument.class);
     Mockito.when(sr.getId()).thenReturn(1L);
-    rec.setInstrument(sr);
-    
+    instrumentService.addServiceRecord(rec, sr);
```

Comment:
Remove. Service Record is already added to the instrument, and is re-saved below, which is what we're testing. Can remove the Instrument mock above too since it's no longer used